pkg/volume/nfs/nfs.go

```diff
@@ -61,7 +62,8 @@ var _ volume.PersistentVolumePlugin = &nfsPlugin{}
 var _ volume.RecyclableVolumePlugin = &nfsPlugin{}
 
 const (
-	nfsPluginName = "kubernetes.io/nfs"
+	nfsPluginName  = "kubernetes.io/nfs"
+	unMountTimeout = time.Minute
```

Comment:
just want to double check whether this is optimal timeout setting in this case..

Reply:
there is no 100% sure way to know the answer. We have deployed some CSI driver with this value and it appears to be working fine. Unlike block devices - NFS volumes don't have to fsync on umount, so I think it should be fine.